packages/rest-api-client/package.json

```
@@ -78,5 +79,10 @@
     "form-data": "^3.0.0",
     "js-base64": "^3.4.5",
     "qs": "^6.9.4"
+  },
+  "exports": {
+    "import": "./index.mjs",
+    "require": "./lib/index.js",
+    "default": "./lib/index.js"
```

webpack v5 will see "exports" field.

[add exportsFields and conditionNames options by vankop · Pull Request #10953 · webpack/webpack](https://github.com/webpack/webpack/pull/10953)

```suggestion
    "node": {
      "import": "./index.mjs",
      "require": "./lib/index.js",
      "default": "./lib/index.js"
    },
    "browser": "lib/index.browser.js"
```